go.mod

```diff
@@ -414,6 +414,12 @@ replace (
 	// https://github.com/zeta-chain/tss-lib/tree/threshold-dep-updates
 	// which is a fork of https://github.com/threshold-network/tss-lib
 	github.com/bnb-chain/tss-lib => github.com/zeta-chain/tss-lib v0.0.0-20240916163010-2e6b438bd901
-	github.com/ethereum/go-ethereum => github.com/zeta-chain/go-ethereum v1.13.16-0.20241022183758-422c6ef93ccc
+
+	github.com/cosmos/evm => github.com/skosito/evm v0.0.0-20250716000216-07614a1d33ef
+
+	github.com/cosmos/evm/evmd => github.com/skosito/evm/evmd v0.0.0-20250716000216-07614a1d33ef
+
+	// use Cosmos geth fork
+	github.com/ethereum/go-ethereum => github.com/cosmos/go-ethereum v1.15.11-cosmos-0
```

Comment:
Maybe we can just remove this replace and directly use cosmos/go-ethereum?

Can be done in another PR